src/instance/migrate.jl

```diff
@@ -31,7 +31,7 @@ function _migrate_to_v03(json)
         json["Reserves"]["r1"]["Type"] = "spinning"
         json["Reserves"]["r1"]["Amount (MW)"] = amount
         for (gen_name, gen) in json["Generators"]
-            if gen["Provides spinning reserves?"] == true
```

Comment:
Actually, on a closer look, could you clarify how is this change supposed to fix the problem? `x == true` and `x !== false` are almost equivalent. A failing test case would be helpful.

Reply:
Hi Alinson, sure. Since ``"Provides spinning reserves?"`` is by default true for generators, json files for scenarios/instances may omit this field. In such a case, the ``_migrate_to_v03`` would assume that the generator does not provide spinning reserves, as it does not see the ``"Provides spinning reserves?"`` key explicitly having a ``true`` value. This change ensures that, unless otherwise explicitly specified, a generator provides spinning reserves.